### ISSUE TITLE
The color contrast ratio for the "Azure OpenAI link" and its surrounding text is below 4.5:1, which causes it to fail the accessibility requirement: A11y_TypeChat_Docs Link_Example screen_Contrast(Minimum)

### ISSUE THREAD

**Issue by keerthiduvvuri** [microsoft/TypeChat#279]
[[Check out Accessibility Insights!](https://accessibilityinsights.io/)]- Identify accessibility bugs before check-in and make bug fixing faster and easier.

### GitHubTags:
#A11yTCS; #A11ySev2; #TypeChat; #BM_TypeChat_Web_Nov2025; #Win11; #Contrast(Minimum); #WCAG1.4.3; #FTP; #DesktopWeb; #A11yMAS; #ChromiumEdge;.

### Environment and OS details:
URL: https://microsoft.github.io/TypeChat/
Windows 11
OS build: 26200.6899
Chromium Edge Version 142.0.3595.53 (Official build) (64-bit)

### Repro Steps:

1. Hit the URL: https://microsoft.github.io/TypeChat/
2. Typechat Home screen gets opened.
3. Tab Navigate to "Doc Link".
4. Introduction page gets opened.
5. Tab navigate to examples link and press enter on it.
6. Examples screen gets opened
7. Turn on Accessibility insights and Observe that The color contrast ratio for the "Azure OpenAI link" and its surrounding text is below 4.5:1, which causes it to fail the accessibility requirement.

### Actual Result:
The color contrast ratio for the "Azure OpenAI link" and its surrounding text is below 4.5:1, which causes it to fail the accessibility requirement.

### Expected Result:
The color contrast ratio for the "Azure OpenAI link" and its surrounding text should be greater than 4.5:1.

### User Impact:
When the color contrast ratio for the "Azure OpenAI link" and its surrounding text falls below 4.5:1, it does not meet accessibility standards. This can make it challenging for users with low vision to read or interact with the link and its nearby text, potentially limiting their access to important information or features. Ensuring a sufficient contrast ratio is crucial to provide an inclusive experience for all users and comply with accessibility guidelines.

### Attachment:

<img width="960" height="540" alt="Image" src="https://github.com/user-attachments/assets/5b5abe22-f3e6-4327-b91e-aecea6ef242a" />

**Comment by DanielRosenwasser** [comment]
This is not accurate, the background color is white (#FFFFFF) not black/dark gray (#212529). Looks like the accessibility tool is comparing the surrounding text color with the styled text color.
